canonical-web-and-design/vanilla-framework
# Added a node.js script that creates a class reference yaml

## Done

- Added a node.js script that creates yaml 

## QA

- Install dependencies with `yarn`
- Run `yarn create-class-references` command
- Verify "build/classreferences.yaml" is created
- Verify it contains notification and side-navigation components

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

[if relevant, include a screenshot or screen capture]
